### ISSUE TITLE
Ordered bullets and script tags together do not support empty lines

### ISSUE THREAD

**Issue by openchuep** [simov/markdown-viewer#140]
The following breaks the rendering as soon as there is an empty line in between the ```.

1.  python 3.7.9:
```shell
Some code

Here
```

**Comment by simov** [comment]
Yes, Remark does not render this case correctly, but Marked does - you can switch to it from the compiler tab. Additionally Markdown Viewer supports a few others when installed locally https://github.com/simov/markdown-viewer#more-compilers
